commands/test/test-command.ts

```diff
@@ -286,12 +291,12 @@ async function findMatchingUser(userId: string, userResponses: number[]): Promis
 }
 
 function verifyUser(interaction: any) {
-    const guild: Guild | undefined = client.guilds.cache.get('1131613084553859182');
+    const guild: Guild | undefined = client.guilds.cache.get('1119231777391788062');
```

Comment:
We already store the guild id in our env file

Reply:
is there a way we can import it into the code here? Or should the old Guild id be used here?